contentcuration/contentcuration/frontend/shared/views/channel/ChannelDetailsModal.vue

```diff
@@ -92,16 +83,15 @@
       },
       backLink() {
         return {
-          name: this.$route.matched[0].name,
-          query: this.$route.query,
-          params: {
-            ...this.$route.params,
-            channelId: null,
-          },
+          name: this.$route.query.last,
```

Comment:
Following up on my other comment, just need to make sure the query params are still available here

Reply:
Done, thanks for catching this